When running ./configure.py --bootstrap, do not automatically run ./ninja
--- configure.py.orig	2024-05-11 06:43:36.000000000 -0500
+++ configure.py	2024-05-17 00:15:59.000000000 -0500
@@ -675,28 +675,3 @@
 n.close()
 print('wrote %s.' % BUILD_FILENAME)
 
-if options.bootstrap:
-    print('bootstrap complete.  rebuilding...')
-
-    rebuild_args = []
-
-    if platform.can_rebuild_in_place():
-        rebuild_args.append('./ninja')
-    else:
-        if platform.is_windows():
-            bootstrap_exe = 'ninja.bootstrap.exe'
-            final_exe = 'ninja.exe'
-        else:
-            bootstrap_exe = './ninja.bootstrap'
-            final_exe = './ninja'
-
-        if os.path.exists(bootstrap_exe):
-            os.unlink(bootstrap_exe)
-        os.rename(final_exe, bootstrap_exe)
-
-        rebuild_args.append(bootstrap_exe)
-
-    if options.verbose:
-        rebuild_args.append('-v')
-
-    subprocess.check_call(rebuild_args)
